apps/web/app/future/[user]/[type]/page.tsx

```diff
@@ -28,6 +28,7 @@ export const generateMetadata = async ({ params, searchParams }: PageProps) => {
     isTeamEvent: false,
     org: isValidOrgDomain ? currentOrgDomain : null,
     fromRedirectOfNonOrgLink: legacyCtx.query.orgRedirection === "true",
+    limitHostsToThree: true,
```

Comment:
This is User  event booking page. Is it okay to limit the hosts in metadata?

Reply:
not required here